lib/index.js

```diff
@@ -1167,10 +1167,25 @@ module.exports = (function() {
             }
             return ab;
         };
-        substitution = self.imageToBuffer(substitution)
+        try {
+            substitution = self.imageToBuffer(substitution)
+        } catch (error) {
+            console.error("Error on reading image object : ", error)
+            return false;
+        }
+        if( substitution == null){
```

Comment:
```suggestion
        if (substitution == null) {
```

Reply:
ok, done.
I think I will make an another branch for fix all my formatting mistake.